.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+on: ["push", "pull_request"]
+name: Unit tests
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.13.x, 1.14.x, 1.15.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    env:
+      GO111MODULE: off
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+    - name: Setup GOPATH
+      run: |
+        gopath_org=$(go env GOPATH)/src/github.com/intel/
+        mkdir -p ${gopath_org}
+        ln -s ${PWD} ${gopath_org}
```

Comment:
Why do you need this?

Reply:
yes, we need it, otherwise `goveralls` will fail because it's not in the GOPATH (go test .../.)

Reply:
see https://github.com/devimc/govmm/actions/runs/212445829